lib/plugins/aws/package/compile/functions/index.js

```diff
@@ -182,6 +182,39 @@ class AwsCompileFunctions {
       }
     }
 
+    if (functionObject.awsKmsKeyArn) {
```

Comment:
I think for consistency it would be better to rename `awsKmsKeyArn` as `kmsKeyArn`